build-logic/publication/src/main/kotlin/convention.sonatype.gradle.kts

```diff
@@ -77,7 +77,7 @@ signing {
         .orNull
     val signingKey = providers.gradleProperty("kaspresso.pgp.key")
         .forUseAtConfigurationTime()
-        .orNull?.toFile()?.readText()
```

Comment:
@Fullik here it was possible to leave work with a file, not text
Perhaps it would be even more convenient.

Reply:
We cant pass to github secrets a file, only string. To prevent any security issues with writing key into the file we have to use this property as text.